### PR TITLE
Portfolio desktop version

### PR DESCRIPTION
**Pull request by jugosack**
Dear Reviewer!
Please find activities towards the general requirements:

- there are no linter errors](https://github.com/microverseinc/linters-config).
- It is used correct [GitHub flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md).
- Work is documented [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).

HTML/CSS requirements

- Followed is yours list of [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).

Project requirements
- You need to make sure that your dekstop version looks like the one in the Figma design template that you chose. 
- Use media queries in your CSS file to make your desktop version look like the template you selected (Desktop: 768px or wider).

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -927,13 +935,37 @@ body {
...
+    /*
+width: 90%;
+  font-weight: 700;
+  font-size: 40px;
+  margin-bottom: -1%;
+  text-align: center;
+  color: #fff;
+  padding: 70px 10px 20px 10px;
+  */
+  }
```
Comment (roshan-bajgain) [inline style.css]: - [ ] [Optional] It would be nicer to remove the commented code because it makes the code unclean.

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @jugosack, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️

## Required Changes ♻️
- [ ] I cannot leave a comment there so I am leaving it here. Please implement it.
- [ ] According to the project requirement, you should stick to the design as much as possible like color, font, background, and spacing. If you see the content of each card then you will find it different. So please stick to the design
- [ ] Everything is good but the small chance you need to make is a gap between the list. please see the image below for reference.
- [ ] I notice every card has the same issue so please implement it in all the cards
### Your design
your
### Figma design
fimga

- [ ] Your design is great but you need to make a small change, please implement it. You should stick to the design as much as possible like Font, color, and background. I notice that you have missed the small design on the background. please implement it. You can download it one by one and use it. please see the image below for reference
### Your design
your
### Figma design

Screenshot 2022-12-08 182057


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (jugosack) [reply style.css]: @roshan-bajgain Thanks for your comments and suggestions. 

- I have improved the code to your directions. 

Screen Shot 2022-12-08 at 4 44 34 PM

- Also, I have created new background.

Screen Shot 2022-12-08 at 4 46 27 PM

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @jugosack, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - No linter errors. ✔️
 - GitHub flow is followed. ✔️
 - Good UI design and transitions. ✔️ 

## Aspects to improve ♻️

- [x] Check the comments under the review.
- [x] As the previous Code Reviewer mentioned, there is a small issue with the project cards list of technologies gap. There is still no gap between the list items. Kindly add a gap to match the Figma template.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (jugosack) [reply style.css]: @leonardodiasb Thanks for your comments and suggestions. I have improved the code to your directions, where the gap is created among technlogies (css, html javascript).

Screen Shot 2022-12-08 at 7 26 49 PM

**Review by Gopxfs — APPROVED** [review]
# 🟢 STATUS: APPROVED 🟢

## Hello @jugosack 👋,

Your project is complete! You followed all the requirements, feel free to merge it! 😄
Congratulations! 🎉 



## Highlights
✔️ Linters are passing successfully.
✔️ Implemented all the changes requested by the previous reviewer. Well done!

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**